src/fonts.c

```diff
@@ -112,3 +112,710 @@ uint8_t digits[10][8]={
       0b00000000,
     }
 };
+
+

```

Comment:
Agregaria una tabla de referencia para entender bien que representa cada elemento, asi es mas facil entender y calcular los offset segun tipo de char
```suggestion
/*
0-9 : {0-9}
10 : :
11 : !
12 : ?
13 : ¿
14 : ¡
15 : (space)
16 : (space 2)?
17 - 69 : A-Z, a-z
*/
```